main.py

```diff
@@ -0,0 +1,79 @@
+#   Domato - main script
+#   --------------------------------------
+#
+#   Written and maintained by Ivan Fratric <ifratric@google.com>
+#
+#   Copyright 2017 Google Inc. All Rights Reserved.
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+
+import generator
+import argparse
+import os
+
+def get_parser():
```

Comment:
please rename to `get_argument_parser` as "parser" in Domato can mean many things.